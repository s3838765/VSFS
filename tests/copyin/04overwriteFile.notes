NOTES V1.0
=copyin/
@copyin/04overwriteFile
 This file already exists internally!
 When the copyin command is used...
 This file will be overwritten :(
