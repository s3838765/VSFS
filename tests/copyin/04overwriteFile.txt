This is the replaced file!
I didn't like the original file!!
I WILL TAKE OVER THE WORLD OF FILES!!!!!!!
