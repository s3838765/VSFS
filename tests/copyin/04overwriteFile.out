NOTES V1.0
=copyin/
#copyin/04overwriteFile
#This file already exists internally!
#When the copyin command is used...
#This file will be overwritten :(
@copyin/04overwriteFile
 This is the replaced file!
 I didn't like the original file!!
 I WILL TAKE OVER THE WORLD OF FILES!!!!!!!
